src/ResourceManager/Insights/Commands.Insights/Diagnostics/SetAzureRmDiagnosticSettingCommand.cs

```diff
@@ -34,6 +33,7 @@ public class SetAzureRmDiagnosticSettingCommand : ManagementCmdletBase
     {
         public const string StorageAccountIdParamName = "StorageAccountId";
         public const string ServiceBusRuleIdParamName = "ServiceBusRuleId";
+        public const string EventHubRuleIdParamName = "EventHubAuthorizationRuleId";
```

Comment:
This variable is being used by Vinicius to check if the parameter EventHubAuthorizationRuleId is actually being sent in the call as opposed to it having a null value because the user did not include it in the call.
Since EventHubAuthorizationRuleId is a string, we do not know if it is null because it was not present in the call (in that case we do nothing to the attribute) or it was null because the user wants us to set a null value in that attribute (in this case we set the attribute/field to null.)